src/main/resources/entity-types/inventory/simple_holdings_records.json5

```diff
@@ -214,7 +214,9 @@
       queryable: true,
       visibleByDefault: false,
       valueGetter: ":sourceAlias.jsonb->>'callNumber'",
-      essential: true
+      essential: true,
+      filterValueGetter: "\"left\"(lower(:sourceAlias.jsonb ->> 'call_number'::text), 600)",
```

Comment:
The valueGetter and filterValueGetter are using different jsonb fields here, which seems incorrect